typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -11,103 +11,116 @@ import { Chains, Mainnets, Testnets } from './chains';
 export const avaxToken = { decimals: 18, name: 'Avalanche', symbol: 'AVAX' };
```

Comment:
Sorry for the noise here, the eslint sort-keys fixer reordered a bunch of stuff.
The actual changes are: 
1. Remove NeutronTestnet
2. Remove healthy RPC URLs

Reply:
Probably worth a changeset comment for the SDK